src/components/CreateContact/CreateContact.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react'
 import styled from 'styled-components'
+import { v4 as uuidv4 } from 'uuid';
 
 export default function CreateContact({ onSubmit }) {
   const [disabledButton, setDisabledButton] = useState(true)
```

Comment:
naming could be better. try to find something like isDisabled because it's way easier to now, that this will store a true or false value